app/javascript/components/applications/application_form/PrimaryApplicantSection.js

```diff
@@ -33,15 +65,15 @@ const PrimaryApplicantSection = ({formApi, editValues }) => {
               <div className="form-group">
                 <div className="small-4 columns">
                   <label>First Name <span className="checkbox-block_note no-margin">(required)</span></label>
-                  <Text required="true" field="firstName" />
+                  <Text field="first_name" />
```

Comment:
is field tied to the variable stored in formAPI? do we want to change all of them to snake case?

Reply:
@akegan This is and all the fields are actually in snake case, it's because we do a field mapping from SOQL to ShortForm Custom API and back to stay consistent. So instead of firstName it's first_name, dateOfBirth is date_of_birth, lastName is last_name in the React world. And we have a service that does the back and forth mapping form SOQL<-> ShortForm Custom API

Reply:
other forms that I was looking at have field='firstName' though. Should we update all of them?

Reply:
Oh wait I was totally looking at the wrong thing. I'm looking at the code now and I think you're right and actually caught a bug when you actually submit 👍 Thank for flagging, debugging now 😸